README.md

```diff
@@ -1,7 +1,7 @@
 # python-project-template
 
-Prerequisites: [`cookiecutter`](https://github.com/cookiecutter/cookiecutter)
+Prerequisites: [`cruft`](https://github.com/cruft/cruft)
 
 ```bash
-cookiecutter gh:runtimeverification/python-project-template
+cruft create https://github.com/runtimeverification/python-project-template
 ```
```

Comment:
This is somewhat misleading, as strictly speaking `cruft` is not a prerequisite for the template. So I'd formulate it something like this.

> [`cookiecutter`](https://github.com/cookiecutter/cookiecutter) template for RV's Python projects.
>
> To keep your project up to date with the most recent version of this template, we recommend [`cruft`](https://github.com/cruft/cruft). You can generate your project with

```bash
cruft create https://github.com/runtimeverification/python-project-template
```